content/03.ml-concepts.md

```diff
@@ -1,24 +1,29 @@
-## Tip 1: Concepts that apply to machine learning also apply to deep learning {#concepts}
-Deep learning is a distinct subfield of machine learning, but it is still a subfield.
-DL has proven to be an extremely powerful paradigm capable of outperforming “traditional” machine learning approaches in certain contexts, but it is not immune to the many limitations inherent to machine learning.
-Many best practices for machine learning also apply to deep learning.
-Like all computational methods, deep learning should be applied in a systematic manner that is reproducible and rigorously tested.
+## Tip 1: Successful deep learning applications require best practices for machine learning and model evaluation {#concepts}
```

Comment:
Can we make the title more "punchy"?  Or at least an imperative sentence with an implied "you" as the subject so that it matches all of the other tip titles?

Something like: 
> Follow best practices for machine learning and model evaluation

Reply:
I am not a big fan of the imperative style because it sounds like orders or commandments, but, yeah, we should be consistent! How about

> Use Machine Learning Best Practices for Model Evaluation

we could also use 

> Use Machine Learning Best Practices for Rigorous Model Evaluation

but "rigorous" has a particular meaning in science so I'd prefer the first version.

Reply:
ok changed it now